packages/sources/enzyme/test/integration/calcgav.test.ts

```diff
@@ -0,0 +1,64 @@
+import { AdapterRequest } from '@chainlink/types'
+import request from 'supertest'
+import nock from 'nock'
+import http from 'http'
+import { server as startServer } from '../../src'
+import { mockEthereumResponseSuccess } from './fixtures'
+
+let oldEnv: NodeJS.ProcessEnv
+
+beforeAll(() => {
+  oldEnv = JSON.parse(JSON.stringify(process.env))
```

Comment:
Just curious, why the serializing and deserializing or `process.env`?

Reply:
This is to clone the `process.env` object, instead of making a reference to it. If we didn't do this, any changes that we do to `process.env` would also be made to `oldEnv`, and we wouldn't be able to restore it after the test has run.

Reply:
Makes sense. I've used `Object.assign` or the object spread syntax (`...`) in the past.